### PR TITLE
Fix add new inline on grappelli

### PR DESCRIPTION
**Pull request by marticols**
Capture when the user clicks on the span for adding new inline.
Slice 1

### CODE REVIEW

File: ckeditor/static/ckeditor/ckeditor-init.js [hunk 1]
```diff
@@ -57,7 +57,8 @@
     document.body.addEventListener('click', function(e) {
       if (e.target && (
         e.target.matches('.add-row a') ||
-        e.target.matches('.grp-add-handler')
+        e.target.matches('.grp-add-handler') ||
+        e.target.parentElement.matches('.grp-add-handler')
```
Comment (matthiask) [inline ckeditor/static/ckeditor/ckeditor-init.js:61]: I think `.closest(".grp-add-handler")` might work well here instead. https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Comment (marticols) [reply ckeditor/static/ckeditor/ckeditor-init.js:61]: Yes!

Comment (fdintino) [reply ckeditor/static/ckeditor/ckeditor-init.js:61]: I notice that there's a polyfill for `Element.prototype.matches` above, but `closest` has even less support (caniuse compare: [Element.closest()](https://caniuse.com/#feat=element-closest) vs [matches()](https://caniuse.com/#feat=matchesselector)). Notably, it's not available in any versions of IE.

**Comment by matthiask** [comment]
Closing in favor of #621 .

**Pull request closed without merging**